code/modules/holiday/holidays.dm

```diff
@@ -394,6 +448,20 @@
 		COLOR_PRIDE_ORANGE,
 		COLOR_PRIDE_RED,
 	)
+	holiday_mail = list(
+		/obj/item/clothing/gloves/color/rainbow,
+		/obj/item/toy/crayon/rainbow,
+		/obj/item/food/grown/rainbow_flower,
+		/obj/item/clothing/under/color/rainbow,
+		/obj/item/clothing/shoes/sneakers/rainbow,
+		/obj/item/clothing/under/color/jumpskirt/rainbow,
+		/obj/item/bedsheet/rainbow,
+		/obj/item/food/egg/rainbow,
+		/obj/item/food/snowcones/rainbow,
+		/obj/item/clothing/head/costume/garland/rainbowbunch,
+		/obj/item/clothing/head/soft/rainbow,
+		/obj/item/clothing/accessory/pride
+		)
```

Comment:
```suggestion
		/obj/item/clothing/accessory/pride,
	)
```
As a general comment for multiple places in this file, end your list with a trailing comma and deindent the closing bracket

Reply:
Also preferably alphabetise these lists

Reply:
wtf did I forget one